plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -307,15 +308,6 @@ public static function get_plugin( $slug ) {
 						'value'     => 'US',
 						'operation' => '=',
 					),
-					array(
-						'type'    => 'not',
-						'operand' => array(
-							array(
-								'type'    => 'plugins_activated',
-								'plugins' => array( 'woocommerce-services' ),
-							),
-						),
-					),
 					array(
 						'type'    => 'not',
```

Comment:
Do we need the "is visible if woocommerce-shipping is not activated" condition? The other plugins don't seem like they need to declare that if the plugin is themselves. 

If we don't need it, then we can delete all of them to simplify the conditions.

Reply:
Nope! They weren't necessary 😄 I've removed them from WCS&T and WC Shipping 👏 
The condition still exist for two other plugins (Jetpack and Google for WooCommerce) but I don't want to modify them in this PR since it's a bit outside scope and I don't want to risk something if it's used later by some other hook for the specific plugins 🙈